docker-app/qfieldcloud/core/management/commands/dequeue.py

```diff
@@ -114,13 +114,16 @@ def handle(
             if single_shot:
                 break
 
-    def _run(self, job: Job) -> None:
-        job_run_classes: dict[Job.Type, type[JobRun]] = {
+    def get_job_mapping(self) -> dict[Job.Type, type[JobRun]]:
+        return {
             Job.Type.PACKAGE: PackageJobRun,
-            Job.Type.DELTA_APPLY: DeltaApplyJobRun,
+            Job.Type.DELTA_APPLY: ApplyDeltaJobRun,
```

Comment:
I promise one day we also change the `DELTA_APPLY` name with `APPLY_DELTA`. I am sorry for the dislexic name I introduced back then.